frontend/src/app/shared/components/autocompleter/op-autocompleter/op-autocompleter.component.ts

```diff
@@ -334,6 +334,7 @@ export class OpAutocompleterComponent extends UntilDestroyedMixin implements Aft
   }
 
   public keydowned(val:any) {
+    this.repositionDropdown();
```

Comment:
I'd avoid repositioning the dropdown **every** time we type because it has a timeout included but no debouncing.